archivy/templates/dataobjs/show.html

```diff
@@ -17,36 +17,55 @@
     		<ul id="searchHits"></ul>
 		</div>
 	{% endif %}
-    <h2 id="post-title">{{ dataobj['title'] }}</h2>
-    {{ ", ".join(dataobj['tags']) }}
-    <br>
+  <div class="post-header">
+    <!-- Form for editing the frontmatter -->
+    {% if not view_only %}
+    <form id="post-title-form">
+      {{ post_title_form.hidden_tag() }}
+        {{ post_title_form.title(**{"aria-describedby":"username-input-validation"}) }}
```

Comment:
Could you explain what the `aria-describedby` here does?

Reply:
Ah sorry I was copying and pasting.